src/test/component.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+
+import CalculatorContent from '../components/calculator/CalculatorContent';
+import HomeContent from '../components/home/HomeContent';
+import QuoteContent from '../components/quotes/QuoteContent';
+import Layout from '../components/layout/Layout';
+
+describe('testing components', () => {
+  it('renders Calculator component correctly', () => {
+    const tree = renderer.create(<CalculatorContent />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Home Component correctly', () => {
+    const tree = renderer.create(<HomeContent />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Quote component correctly', () => {
+    const tree = renderer.create(<QuoteContent />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('full app rendering/navigating', async () => {
+    render(<Layout />, { wrapper: BrowserRouter });
+    expect(screen.queryByText(/Home/)).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job @team. You did an impressive job already. One major thing left is for you to use [React Testing Library](https://github.com/testing-library/react-testing-library) to simulate user interaction. 
> Remember: this is different from `snapshots` tests. While `snapshots` tests are awesome to follow up when the UI changes the `React Testing Library` is used to simulate the user interaction. 
### Some examples would be 
```JavaScript
expect(screen.queryByText(testMessage)).toBeNull()
fireEvent.click(screen.getByLabelText(/show/i))
expect(screen.getByText(testMessage)).toBeInTheDocument()
```

Reply:
Thanks for the review and suggestion @Tadesse-Alemayehu.

Reply:
Good job making the changes @Trust-007 👌